addons/medical_ai/StateMachine.hpp

```diff
@@ -1,5 +1,5 @@
 class GVAR(stateMachine) {
-    list = "allUnits select {local _x}";
+    list = "allUnits select {local _x && {_x != ACE_player} && {!isPlayer _x}}";
```

Comment:
Zeus remote controled AI return false for `isPlayer`
could also use `ace_common_fnc_isPlayer`